lib/src/appodeal.dart

```diff
@@ -189,5 +189,16 @@ class Appodeal {
     await fetchConsentInfo();
     return _channel.invokeMethod('requestConsentAuthorization');
   }
+
+  // endregion
+
+  // region - Permissions
```

Comment:
I see that you did this change for the Android integration only, but do you also have experience with iOS? If yes, then it would great if you could do this change on the iOS code base as well so both platforms are supported 🙂